common/conf/env/CODE.properties

```diff
@@ -25,6 +25,7 @@ oas.siteId.host=m.code.dev-theguardian.com
 
 amp.host=amp.code.dev-theguardian.com
 amp.scheme=https://
+amp.cors.origin=https://m.code.dev-theguardian.com,https://amp.code.dev-theguardian.com, https://cdn.ampproject.org
```

Comment:
I would rather keep consistent with what we have had before: https://github.com/guardian/frontend/blob/credentials-amp-list/common/conf/env/CODE.properties#L22

Also, only one every gets used at once, so in a way they are just all options for one origin.